sql/2020/22_HTTP_2/number_of_h2_and_h3_pushed_resources_and_bytes_by_content_type.sql

```diff
@@ -0,0 +1,41 @@
+# standardSQL
+# Number of H2 and H3 Pushed Resources and Bytes by Content type
+SELECT 
+  percentile,
+  client,
+  http_version,
+  content_type,
+  COUNT(DISTINCT page) AS num_pages,
+  APPROX_QUANTILES(num_requests, 1000)[OFFSET(500)] AS pushed_requests,
+  APPROX_QUANTILES(kb_transfered, 1000)[OFFSET(500)] AS kb_transfered,
```

Comment:
Use the percentiles :)
```suggestion
  APPROX_QUANTILES(num_requests, 1000)[OFFSET(percentile * 10)] AS pushed_requests,
  APPROX_QUANTILES(kb_transfered, 1000)[OFFSET(percentile * 10)] AS kb_transfered,
```